src/ngraph_builder.cc

```diff
@@ -33,6 +33,10 @@
 #include "tensorflow/core/graph/edgeset.h"
 #include "tensorflow/core/lib/core/errors.h"
 
+#if defined(NGRAPH_DISTRIBUTED)
+#include <mpi.h>
```

Comment:
please use distributed class and get rank in this file.

Reply:
changed

Reply:
you don't have to say. Github shows it's `Outdated` after change on the line